Framework/DataHandling/test/LoadPSIMuonBinTest.h

```diff
@@ -152,9 +152,9 @@ class LoadPSIMuonBinTest : public CxxTest::TestSuite {
     double lastGoodData = std::stod(alg.getPropertyValue("LastGoodData"));
     double timeZero = std::stod(alg.getPropertyValue("TimeZero"));
 
-    TS_ASSERT_DELTA(firstGoodData, 0.167, 0.001);
```

Comment:
Why did this value change in the tests?

Reply:
Changed it by mistake and then changed it back.